content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+**Note:** This documentation describes how to configure CI to assist with Bugzilla-related processes. It does _not_ describe
```

Comment:
Again, I really don't think this is the (only) doc we want or need. The point of this doc is to alleviate questions from help-desk. Questions do not come into helpdesk about "how does `valid_states` get used to validate my PR?".

I think we should focus on writing a doc that fulfills this user story from the Jira card:

As a author of a PR to an OpenShift component
I want to understand how my PR may interact with Bugzilla, what the normal workflows are and what I need to keep in mind
So that I can be effective at fixing bugs with pull requests

So, as an example, instead of this preamble, I'd perhaps write a sentence like:

> Many automated processes exist to help you navigate the OpenShift Bugzilla workflow when you're fixing a Bugzilla bug with a GitHub Pull Request. Read up on the overall process [here](...) before diving into the specifics of automation.

Reply:
Is this a call to inline the current backport-process Google Doc?  Because I'm in favor of Git over Google Docs for version control, but I'm not sure if everyone else is on board with that or not :p.

Reply:
Never mind, I see the linked Source page [linking to Google Docs in turn][1].

[1]: https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/openshift_bugzilla_process#jive_content_id_Fixing_in_multiple_releases